compiler/rustc_expand/src/base.rs

```diff
@@ -855,6 +856,25 @@ impl SyntaxExtension {
     }
 }
 
+/// Built-in derives
+/// Only derives that are being used for special casing expansion are represented here
+#[derive(Debug, Copy, Clone, Hash, Eq, PartialEq)]
+pub enum BuiltinDerive {
+    Copy,
+    PartialEq,
+}
```

Comment:
We actually had this enum in the past, but it was represented as bitflags.
See https://github.com/rust-lang/rust/pull/65892 where it was removed.

Reply:
Oh wow, I didn't realize! I was considering using bitflags, but I didn't want to add a dependency to `rustc_expand`. I can make this change if we don't close the PR.